recipes/jupyterlab-sos/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterlab-sos" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://registry.npmjs.org/jupyterlab-sos/-/jupyterlab-sos-{{ version }}.tgz
+  sha256: afbfa0c906a45db6e97b12731d308de89e80990833401355198f75f455328de5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - nodejs
```

Comment:
Should this be in host?

Reply:
I do not know. All other jupyterlab extensions have this in build https://github.com/conda-forge/jupyterlab-git-feedstock/blob/master/recipe/meta.yaml is one of them.

Reply:
it should be `build` as it is a build dependency and not a host one.

Reply:
kept in `build`.